### PR TITLE
Update primary nav

### LINKED ISSUE #2300: Nav Update: Primary Nav on white
Followup from #1727 and #1721 and comps https://redpen.io/p/avd5fc0b3db14a7883

### Main Nav
- [ ] Put nav on white background everywhere (already have this on Home, Initiatives, Opportunity/Campaign, Participate, Fellowships pages but need to add this to the rest of pages **Internet Health, People, About Us, News**)

- [ ] Update all primary nav selected state: increase thickness of the black line to 6px and move it lower down so it is on top of the grey line

image

### Overview
nav new 2x

### PR DESCRIPTION
**Pull request by alanmoo**
~~First section of #1721~~

Fixed #2300

### CODE REVIEW

**Comment by Pomax** [comment]
@alanmoo can we close this?

**Comment by alanmoo** [comment]
I believe @mmmavis is going to pick up that original ticket this week, I'll let her make the call if she wants to use this or not.

**Comment by mmmavis** [comment]
I'll push a few changes to this PR and have Kristina review design before I assign someone for code review.

**Comment by mmmavis** [comment]
@alanmoo once this passes design review I think you can just review that last few commits that I added to this PR

**Comment by kristinashu** [comment]
Nav is on white everywhere which is great! 

## Banner
Not sure if it is from this ticket but the banner looks too short now. Screen cap from PR:

screen shot 2018-12-17 at 9 48 34 am

## Mobile Underline
Also, can the underline but the same length as the page title on mobile too? And bring the line up a little closer to the title? Screen cap from PR:

screen shot 2018-12-17 at 9 49 23 am

## Mobile Donate Button
And can we keep the same Donate button as we have on prod (it's slightly smaller)? Screen cap from prod:
screen shot 2018-12-17 at 9 49 37 am

**Comment by mmmavis** [comment]
@kristinashu

> Not sure if it is from this ticket but the banner looks too short now. 

Oh this is not from this PR. Review app's participate page is using the old participate page model (production and staging are using the updated participate page model). There's already a ticket for it: https://github.com/mozilla/foundation.mozilla.org/issues/1906 So I think we are good here.

**Comment by mmmavis** [comment]
I'll update the 2 other things you mentioned.

**Comment by mmmavis** [comment]
@kristinashu review app is ready for design review!

**Comment by mmmavis** [comment]
tagging @alanmoo for code review

File: source/sass/components/nav.scss [hunk 1]
```diff
@@ -76,11 +75,48 @@
...
-      @media (max-width: $bp-lg) {
-        padding-top: 0.6rem;
+      @at-root {
+        #view-people .nav-link-people,
+        #view-initiatives .nav-link-initiatives,
+        #view-projects .nav-link-projects,
+        #view-participate .nav-link-participate,
+        #view-internet-health .nav-link-internet-health,
+        #view-about .nav-link-about {
+          border-color: black !important;
```
Comment (alanmoo) [inline source/sass/components/nav.scss]: This whole section is weird (though I know it's been here). The `important`s feel really wrong though. Can we remove them if we make line 76 not as nested, and make it so the selector is `#primary-nav-container [class^="nav-link-"]`?

File: source/sass/components/nav.scss [hunk 2]
```diff
@@ -219,16 +255,11 @@
         a {
```
Comment (alanmoo) [inline source/sass/components/nav.scss]: Similar to another comment, can this be `[class^="nav-link-"]` so we're not targeting specific element types with hyper-specificity?

File: source/sass/components/nav.scss (hunk 1 above)
Comment (mmmavis) [reply source/sass/components/nav.scss]: @alanmoo I'll create a followup ticket to do this. There are too many elements using the class `nav-link` so I don't wanna tackle this in this PR just yet. 

I can include this work as part of https://github.com/mozilla/foundation.mozilla.org/pull/2348 or work on it after https://github.com/mozilla/foundation.mozilla.org/pull/2348 is merged.

Comment (mmmavis) [reply source/sass/components/nav.scss]: Follow-up tickets created: https://github.com/mozilla/foundation.mozilla.org/issues/2360

File: source/sass/components/nav.scss (hunk 2 above)
Comment (mmmavis) [reply source/sass/components/nav.scss]: This got cleaned up a bit in https://github.com/mozilla/foundation.mozilla.org/pull/2348 so I'll revisit this as followup.

**Pull request merged**